lib/ores_api.rb

```diff
@@ -41,7 +41,12 @@ class InvalidProjectError < StandardError
   private
 
   def query_url(rev_ids)
-    base_url = "/v3/scores/#{@project_code}/?models=articlequality&features&revids="
+    base_url =
```

Comment:
Instead of an `if/else` here, I think it would make sense to set the model at the same time you set the project code, and then interpolate it here.

Reply:
I have added another variable named `project_model` just like `project_code` for a different model